### PR TITLE
Added my information to the student file and added an image

### PR DESCRIPTION
**Pull request by Srijanjha**

### CODE REVIEW

**Comment by animeshsinghweb** [comment]
@SrijanJha28 There is a merge conflict in the file `students.yml` you need to resolve it first.
You can check out `contributing.md` for more info on that.

**Comment by Srijanjha** [comment]
Edited the file

**Review by abishekvashok — CHANGES_REQUESTED** [review]
That's an unwanted space before the bio.

**Comment by Srijanjha** [comment]
So should I remove the space before bio?

**Comment by animeshsinghweb** [comment]
@SrijanJha28 Also conflicts are not yet solved.

**Comment by abishekvashok** [comment]
@SrijanJha28 yes remove the space if you can't resolve the conflicts try again following the tutorial if it does not help I will help you if it too doesn't work I will resolve for you.

**Comment by ankitrgadiya** [comment]
@SrijanJha28 There is no need to have your picture in images folder, we just need one copy in /images/students/ folder. Kindly remove your extra photo in images folder. :-)

**Comment by Srijanjha** [comment]
done please review it

File: _data/students.yml [hunk 1]
```diff
@@ -400,3 +400,9 @@
   image: zahari.jpg
   facebook: zari.asparuhov
   bio: 17, Bulgaria, web developing and web servers with NodeJS
+
+- name: Srijan Jha
+  github: SrijanJha28
+  image: Srijan Jha.jpg
```
Comment (animeshsinghweb) [inline _data/students.yml]: Please don't use space in the URL. Please also edit  your image file name and remove the space,

**Comment by Srijanjha** [comment]
Done as suggested

**Pull request closed without merging**